index.html

```diff
@@ -0,0 +1,1729 @@
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <div class="us-img">
+                      <img src="resources/mega-millions.jpg" alt="" />
+                    </div>
+                    <h4 class="h5">
+                      <a href="#">Mega Millions Results, Numbers for 5/29/20</a>
+                    </h4>
+                  </div>
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <div class="us-img">
+                      <img src="resources/louisville-protests.jpg" alt="" />
+                    </div>
+                    <h4 class="h5">
+                      <a href="#"
+                        >Police In Louisville Fire Pepper Balls at Reporter and
+                        News Crew on Live TV</a
+                      >
+                    </h4>
+                  </div>
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <article class="d-flex pb-2 border-bottom">
+                      <div class="photo-1">
+                        <img src="resources/news-coronavirus.jpg" alt="" />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >More Democrats Than Republicans Think Coronavirus
+                          Coverage Accurate: Poll</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2 pb-2 border-bottom">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/george-floyd-protest.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >George Floyd Protesters Smash Windows at CNN Center
+                          in Atlanta</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/contact-tracing.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >Texas Senator Tells Governor to End 'Morally Wrong'
+                          Contact Tracing Program</a
+                        >
+                      </h4>
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="section-pick-1 border-top p-3 container-fluid">
+              <div class="container-fluid">
+                <h1 class="block-title--1 text-capitalize">
+                  <a href="#">World</a>
```

Comment:
- [ ] Kindly check this section for medium devices.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83581263-f01ae200-a503-11ea-9e35-418d3cb85054.png) | ![image](https://user-images.githubusercontent.com/57414789/83581233-e2fdf300-a503-11ea-8e58-91fde2af2ed4.png)